dot/telemetry/mailer.go

```diff
@@ -78,6 +78,11 @@ func BootstrapMailer(ctx context.Context, conns []*genesis.TelemetryEndpoint, en
 				}
 			}
 
+			err = response.Body.Close()
+			if err != nil {
+				mailer.logger.Errorf("cannot close body of response from %s: %s", v.Endpoint, err)
```

Comment:
does this need to be an error log? it doesn't really seem to matter if we were ignoring it before, change to debug perhaps?

Reply:
Changed it to warn because:

- I don't think this ever happens, I can't find any documentation/reason why the body close would fail on an http response
- The user can control the log level of the telemetry individually
- It's not really an error since execution continues
- It might happen i.e. if we run out of memory or some catastrophic situation perhaps, so using warn seems appropriate